### PR TITLE
Added readme for pruneOFA examples on SQuAD and GLUE

### PR DESCRIPTION
**Pull request by XinyuYe-Intel**
# What does this PR do?

Add readme for pruneOFA examples on SQuAD and GLUE

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by hshen14 — APPROVED** [review]
LGTM

**Comment by hshen14** [comment]
@echarlaix This PR is to add some examples to reproduce the results published in NeurIPS'21 ENLSP paper "Prune Once for All: Sparse Pre-Trained Language Models" (link: https://arxiv.org/abs/2111.05754). Please review. Thanks.

File: examples/neural_compressor/text-classification/README.md [hunk 1]
```diff
@@ -73,6 +73,61 @@ python run_glue.py \
 In order to apply dynamic, static or aware-training quantization, `quantization_approach` must be set to 
 respectively `dynamic`, `static` or `aware_training`.
 
+### Prune Once For All
+
+The following example demonstrate the steps of reproducing Prune Once For All examples results on GLUE tasks.
```
Comment (echarlaix) [inline examples/neural_compressor/text-classification/README.md:78]:
```suggestion
The following example demonstrate the steps of reproducing Prune Once For All examples results on the sst-2 task.
```

File: examples/neural_compressor/text-classification/README.md [hunk 2]
```diff
@@ -73,6 +73,61 @@ python run_glue.py \
...
+<br>
+This examples will take the pre-trained sparse language model and fine tune it on the downstream task. This fine tune pipeline is two staged. For stage 1, the pattern lock pruning and the distillation are applied to fine tune the pre-trained sparse language model. In stage 2, the pattern lock pruning, distillation and quantization aware training are performed simultaneously on the fine tuned model from stage 1 to obtain the quantized model with the same sparsity pattern as the pre-trained sparse language model.
```
Comment (echarlaix) [inline examples/neural_compressor/text-classification/README.md:80]:
```suggestion
This examples will take a pre-trained DistilBERT with a sparsity of 90% and fine-tune it on the downstream task. This fine-tuning process will be decomposed into two steps. During step 1, distillation as well as pattern lock pruning are both applied during fine-tuning. During step 2, quantization aware training will be additionaly applied, to obtain a fine-tuned quantized model with the same sparsity as the pre-trained model.
```

File: examples/neural_compressor/question-answering/README.md [hunk 3]
```diff
@@ -78,6 +78,63 @@ python run_qa.py \
 In order to apply dynamic, static or aware-training quantization, `quantization_approach` must be set to 
 respectively `dynamic`, `static` or `aware_training`.
 
+## Prune Once For All
+
+The following example demonstrate the steps of reproducing Prune Once For All examples results on SQuAD dataset.
```
Comment (echarlaix) [inline examples/neural_compressor/question-answering/README.md:83]:
```suggestion
The following example demonstrate the steps of reproducing Prune Once For All examples results on SQuAD1.0 dataset.
```

File: examples/neural_compressor/question-answering/README.md [hunk 4]
```diff
@@ -78,6 +78,63 @@ python run_qa.py \
...
+<br>
+This examples will take the pre-trained sparse language model and fine tune it on the downstream task. This fine tune pipeline is two staged. For stage 1, the pattern lock pruning and the distillation are applied to fine tune the pre-trained sparse language model. In stage 2, the pattern lock pruning, distillation and quantization aware training are performed simultaneously on the fine tuned model from stage 1 to obtain the quantized model with the same sparsity pattern as the pre-trained sparse language model.
```
Comment (echarlaix) [inline examples/neural_compressor/question-answering/README.md:85]:
```suggestion
This examples will take a pre-trained DistilBERT with a sparsity of 90% and fine-tune it on the downstream task. This fine-tuning process will be decomposed into two steps. During step 1, distillation as well as pattern lock pruning are both applied during fine-tuning. During step 2, quantization aware training will be additionaly applied, to obtain a fine-tuned quantized model with the same sparsity as the pre-trained model.
```

**Review by echarlaix — COMMENTED** [review]
Great idea to add prunce once for all it in the README. I added some minor comments concerning the two steps explanation

**Comment by XinyuYe-Intel** [comment]
> Great idea to add prunce once for all it in the README. I added some minor comments concerning the two steps explanation

@echarlaix Thanks for your suggestions! I've modified the readmes accordingly.

**Review by PenghuiCheng — APPROVED** [review]

**Review by echarlaix — APPROVED** [review]
LGTM, thanks a lot @XinyuYe-Intel !

**Pull request merged**